shared-module/displayio/Group.c

```diff
@@ -66,6 +69,26 @@ void common_hal_displayio_group_set_hidden(displayio_group_t *self, bool hidden)
             displayio_group_set_hidden_by_parent(layer, hidden);
             continue;
         }
+        #if CIRCUITPY_VECTORIO
+        layer = mp_obj_cast_to_native_base(
+            self->members->items[i], &vectorio_circle_type);
+        if (layer != MP_OBJ_NULL) {
+            common_hal_vectorio_vector_shape_set_dirty(common_hal_vectorio_circle_get_draw_protocol(layer));
+            continue;
+        }
+        layer = mp_obj_cast_to_native_base(
+            self->members->items[i], &vectorio_rectangle_type);
+        if (layer != MP_OBJ_NULL) {
+            common_hal_vectorio_vector_shape_set_dirty(common_hal_vectorio_rectangle_get_draw_protocol(layer));
+            continue;
+        }
+        layer = mp_obj_cast_to_native_base(
+            self->members->items[i], &vectorio_polygon_type);
+        if (layer != MP_OBJ_NULL) {
+            common_hal_vectorio_vector_shape_set_dirty(common_hal_vectorio_polygon_get_draw_protocol(layer));
+            continue;
+        }
+        #endif
```

Comment:
It may be possible that there is some more type agnostic way to do this instead of having to check for and handle all 3 vectorio types individually like this. 

I'll look into it a bit further but this is the way I figured out how to do it first and it does seem to work this way.